stuffer/s2n_stuffer.c

```diff
@@ -274,6 +274,34 @@ int s2n_stuffer_write_bytes(struct s2n_stuffer *stuffer, const uint8_t * data, c
     return 0;
 }
 
+int s2n_stuffer_writev_bytes(struct s2n_stuffer *stuffer, const struct iovec* iov, int iov_count, size_t offs, size_t size)
+{
+    GUARD(s2n_stuffer_skip_write(stuffer, size));
+
+    void *ptr = stuffer->blob.data + stuffer->write_cursor - size;
```

Comment:
I think `s2n_stuffer_raw_write` should do what you want here

Reply:
ok will change